### ISSUE TITLE
Verticle on Docker container won't stop properly

### ISSUE THREAD

**Issue by Anti-Alias** [eclipse/vert.x#2601]
### Version

* vert.x core:

### Context

Stopping a Verticle using the cli on [this Docker container](https://hub.docker.com/r/vertx/vertx3/) fails. This is because the 'ps' command is not installed which is utilized by the StopCommand class [here](https://github.com/eclipse/vert.x/blob/master/src/main/java/io/vertx/core/impl/launcher/commands/StopCommand.java).

The method terminateLinux() fails because the method pid() returns null.
pid() returns null because it relies on the 'ps' command.

### Steps to reproduce

1. Create a simple Verticle. It should not matter what language its implemented in.
In my case, I wrote my app in Kotlin.
2. Fire up a vertx docker container and add the verticle to it.
2. Run the Verticle using the 'start' command.
3. Try to shut down the Verticle using the 'stop' command.
4. ???
5. Profit!

### Extra

* For my code, I was able to fix the app by adding
```Dockerfile
RUN apt-get update && apt-get install -y procps
```
to my Dockerfile which was using vertx/vertx3 as its base.

**Comment by cescoffier** [comment]
Do you know if the image based on `alpine` is also impacted by this issue?

**Comment by Anti-Alias** [comment]
Not sure about the alpine distributions this is is built off of.
I was able to reproduce this on a Linux machine using just docker.

```bash
# Pulls container into local env
docker pull vertx/vertx3

# Runs in detached mode so it doesn't die
docker run -it -d vertx/vertx3

# Enters container. Use id from previous command.
docker exec -it <container_id> bash

# Results in 'bash: ps: command not found'
ps
```

**Comment by cescoffier** [comment]
I've checked that both images have the `ps` command installed (how crazy is that?). Let me know if you see something missing.

**Comment by Anti-Alias** [comment]
Sorry, I must have misunderstood. I see the recent commit you made to the Dockerfile.
d16e9cba3a4275db80e950b2e75a92854a484443

**Comment by cescoffier** [comment]
Yes, I've fixed the "regular" image. I've tried the alpine and `ps` was provided. Do you see something different?

**Comment by Anti-Alias** [comment]
I'll have to check that when I get home, though from what you're describing it should be fine. Thanks for the assist. Perhaps the command is just missing from the non-alpine openjdk images. Bleh.

**Comment by Anti-Alias** [comment]
I also did not try the alpine images myself. Should have clarified.

**Comment by Anti-Alias** [comment]
Docker image openjdk:8u171-jre-alpine has ps command.
openjdk:8u171-jre does not.
Weird...

That's a wrap, I guess.
Thanks again.
